mm2src/coins_activation/src/erc20_token_activation.rs

```diff
@@ -74,7 +74,7 @@ impl TokenActivationOps for EthCoin {
 
     async fn enable_token(
         ticker: String,
-        platform_coin: Self::PlatformCoin,
+        platform_coin: <Self as TokenActivationOps>::PlatformCoin,
```

Comment:
I'm wondering why is this necessary?

Reply:
This trait is required for generic `enable_token` in `token.rs`. So I can use `fn register_token_info` there.

Reply:
I found the reason why it doesn't work as just `Self::PlatformCoin` - there are multiple declaration of the associated type param.

Reply:
Please consider the following changes: https://github.com/KomodoPlatform/atomicDEX-API/commit/5778b89ebe21f9f4b16167ee8d46826ac5657b76
Probably, I'm not right that added `TryPlatformCoinFromMmCoinEnum` and `Clone` trait bounds for `TokenOf::PlatformCoin`

Reply:
We can get rid of `Clone` and use `Arc` for platform coins in tokens. But that's a little bit outside of this task.